velox/exec/trace/test/QueryTraceTest.cpp

```diff
@@ -270,6 +272,9 @@ TEST_F(QueryTracerTest, task) {
     const auto actaulDirs = fs->list(outputDir->getPath());
     ASSERT_EQ(actaulDirs.size(), 1);
     ASSERT_EQ(actaulDirs.at(0), expectedDir);
+    const auto taskIds = getTaskIds(outputDir->getPath(), fs);
```

Comment:
Can you add unit test to check the trace metadata output? thanks!

Reply:
It is covered by the QueryMetadataReader
```C++
auto reader = trace::QueryMetadataReader(outputDir->getPath(), pool());
  reader.read(acutalQueryConfigs, actualConnectorProperties, actualQueryPlan);

  ASSERT_TRUE(isSamePlan(actualQueryPlan, planNode));
  ASSERT_EQ(acutalQueryConfigs.size(), expectedQueryConfigs.size());
  for (const auto& [key, value] : acutalQueryConfigs) {
    ASSERT_EQ(acutalQueryConfigs.at(key), expectedQueryConfigs.at(key));
  }
ASSERT_EQ(
      actualConnectorProperties.size(), expectedConnectorProperties.size());
...
```